src/libs/ReportUtils.js

```diff
@@ -3283,7 +3275,7 @@ function shouldReportBeInOptionList(report, currentReportId, isInGSDMode, betas,
     }
 
     // Include reports that are relevant to the user in any view mode. Criteria include having a draft, having an outstanding IOU, or being assigned to an open task.
-    if (report.hasDraft || isWaitingForIOUActionFromCurrentUser(report) || isWaitingForTaskCompleteFromAssignee(report)) {
+    if (report.hasDraft || shouldShowGBR(report) || isWaitingForTaskCompleteFromAssignee(report)) {
```

Comment:
isWaitingForTaskCompleteFromAssignee is included in shouldShowGBR right? So we don't need it here?

Reply:
Yes, agree, will updated soon!

Reply:
Actually no, this is another function, I didnt touch it, do I need to add it to shouldShowGBR?
<img width="885" alt="image" src="https://github.com/Expensify/App/assets/39777589/a1e47169-e28f-4071-b11c-3f4f8497c3ac">

Reply:
Ah ok, ignore me then.